src/extension/module.js

```diff
@@ -4058,15 +4073,6 @@ import sampleRUM from './rum.js';
             ...getAdminFetchOptions(),
           },
         );
-        // bust client cache for live and production
-        if (config.outerHost) {
-          // reuse purgeURL to ensure page relative paths (e.g. when publishing dependencies)
-          await fetch(`https://${config.outerHost}${purgeURL.pathname}`, { cache: 'reload', mode: 'no-cors' });
-        }
-        if (config.host) {
-          // reuse purgeURL to ensure page relative paths (e.g. when publishing dependencies)
-          await fetch(`https://${config.host}${purgeURL.pathname}`, { cache: 'reload', mode: 'no-cors' });
-        }
```

Comment:
The same can be removed from `update()`: 
https://github.com/adobe/helix-sidekick-extension/pull/731/files#diff-a52dcacf627a0979d7bdc5c7611b3a07f198106007d12a1959d50dbc057354caR3996-R3998

Reply:
You sure? We still need to clear the browser cache when hitting reload since we don't include the cachebuster on reload.

Reply:
You are probably right 😄

Reply:
Ah yes, good call, will remove `this.isEditor()`.

Reply:
I think also `this.isDev()` can be removed.

Reply:
Actually no, the logic needs to be adjusted if `isDev` is true to bust the cache on localhost.